content/kubermatic/master/guides/addons/aws_node_termination_handler/_index.en.md

```diff
@@ -0,0 +1,35 @@
++++
+title = "AWS Node Termination Handler"
+date = 2021-01-27T14:07:00+02:00
+weight = 6
+
++++
+
+## AWS Spot Instance
+
+Kubermatic Kubernetes Platform(KKP) is capable of creating different machine types in more than ten different cloud providers.
+One of those providers is AWS. 
+
+AWS cloud provider supports a special machine instances that are called spot instances. Spot instances(according to aws official 
+documentations) are instances that use spare EC2 capacity that is available for less than the On-Demand price.
```

Comment:
Tiny nit: I think it would sound better to say `documentation` (singular).

```suggestion
documentation) are instances that use spare EC2 capacity that is available for less than the On-Demand price.
```